lib/src/data/repository/genre_remote_repository.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:convert';
+
+import 'package:either_dart/either.dart';
+
+import '../../core/util/constants.dart';
+import '../../core/util/either_string_t.dart';
+import '../../domain/entity/genre_entity.dart';
+import '../../domain/repository/genre_repository_i.dart';
+import '../datasource/mappers/genre_mapper.dart';
+import '../datasource/remote/api_service.dart';
+import '../model/genre_data.dart';
+
+class GenreRemoteRepository implements IGenreRepository {
+  final ApiService apiService;
+
+  GenreRemoteRepository({required this.apiService});
+
+  @override
+  EitherErrorManager<List<Genre>> loadGenres() async {
+    try {
+      final response = await apiService.getGenres();
+      if (response.statusCode == Constants.statusCodeOk) {
```

Comment:
Instead to add a const to 200, you can use HttpStatus.OK

Reply:
fixed